labseu2-workout-tracker/client
# chore: Remove tracker from web app

##  What does this PR do
Remove the word `tracker` from the web app.

##  Description of task to be completed
Have the word tracker not appear on the app.

##  How should this manually be tested
 clone the app, cd into the workout-tracker folder and run `npm install`. 
 run `yarn start` , then check for the word `tracker` if it appears on the app

##  What are the relevant Trello board stories
[ Remove tracker from web app
](https://trello.com/c/RhrFgAJv/122-remove-tracker-from-befit-tracker-title-on-the-landing-page-all-branding-should-be-befit)
##  Screenshots (If appropriate)

<img width="1440" alt="Screenshot 2019-09-03 at 10 31 00 AM" src="https://user-images.githubusercontent.com/19408669/64162039-89fb1480-ce36-11e9-82a0-72fd8cc00a2b.png">


##  Questions (If any):
